<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RealSense</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_frames">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Kernel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sb_kernel">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox{border-width: 1px;border-style: solid; border-color: gray;}QSpinBox::down-button{  width: 25; height: 25;subcontrol-position: left;}QSpinBox::up-button{ width: 25; height: 25;}</string>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cb_dilate">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox::indicator { width:20px; height: 20px;}</string>
         </property>
         <property name="text">
          <string>Enable dilate</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Max distance mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sb_max_d">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox{border-width: 1px;border-style: solid; border-color: gray;}QSpinBox::down-button{  width: 25; height: 25;subcontrol-position: left;}QSpinBox::up-button{ width: 25; height: 25;}</string>
         </property>
         <property name="minimum">
          <number>150</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>350</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Min distance mm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sb_min_d">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox{border-width: 1px;border-style: solid; border-color: gray;}QSpinBox::down-button{  width: 25; height: 25;subcontrol-position: left;}QSpinBox::up-button{ width: 25; height: 25;}</string>
         </property>
         <property name="minimum">
          <number>150</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lb_status">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Disconected</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="rb_640_480">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>640x480, 90fps</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="rb_1280_720">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1280x720, 30fps</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Data ignore ratio %</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="sb_depth_dead_zone">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox{border-width: 1px;border-style: solid; border-color: gray;}QSpinBox::down-button{  width: 25; height: 25;subcontrol-position: left;}QSpinBox::up-button{ width: 25; height: 25;}</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>85</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Min area of work region %</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="sb_min_area_w_region">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox{border-width: 1px;border-style: solid; border-color: gray;}QSpinBox::down-button{  width: 25; height: 25;subcontrol-position: left;}QSpinBox::up-button{ width: 25; height: 25;}</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>80</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Max area of work region %</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="sb_max_area_w_region">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QSpinBox{border-width: 1px;border-style: solid; border-color: gray;}QSpinBox::down-button{  width: 25; height: 25;subcontrol-position: left;}QSpinBox::up-button{ width: 25; height: 25;}</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="pb_start">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(64,238,64)</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="pb_stop">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(240,64,64)</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Latepanda ip</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="le_latepanda_ip">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_28">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Width request port</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="le_width_request_port">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1234</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Video stream dest ip</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="le_dest_stream_ip">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Video stream dest port</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="le_dest_stream_port">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4321</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QPushButton" name="pb_save">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stream status</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="lb_stream_status">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Disable</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QPushButton" name="pb_start_video_stream">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(144,238,144)</string>
         </property>
         <property name="text">
          <string>Start video stream</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QPushButton" name="pb_stop_video_stream">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(240,128,128)</string>
         </property>
         <property name="text">
          <string>Stop video stream</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QPushButton" name="pb_start_width_server">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(144,238,144)</string>
         </property>
         <property name="text">
          <string>Start width get server</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QPushButton" name="pb_stop_width_server">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(240,128,128)</string>
         </property>
         <property name="text">
          <string>Stop width get server</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_29">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Server status</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QLabel" name="lb_width_server_status">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Disable</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="cb_work_region">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox::indicator { width:25px; height: 25px;}</string>
       </property>
       <property name="text">
        <string>Select working region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Width, mm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_width">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
